### ISSUE TITLE
[Feature request] Several Buttons hit on one click with the control mapper

### ISSUE THREAD

**Issue by Jeromebrivet** [SHWotever/SimHub#1710]
I m an user of the control mapper. 
I would like to know if with the plugin or anything else you know a way to bind several buttons hit when you click on a button. 

I have this function on my Rexing mayaris2. 
I chosed to do X3 or x6 with the 6th paddles and i use them for break bias. So if i click left paddle i  get minus X3 click on break bias minus, and if i click the right paddle, i get plus X3 click on break bias plus. Idea is for example on Monza to change repartition between high speed corners and low speed corners.

I would like to get the same function on others wheels i use but i Can t find how to do it, with control mapper or something else. Any Idea perhaps ? Thanks

**Comment by lewisst** [comment]
I already do this in control mapper and bind a single physical button to multiple roles as the same time.

e.g my Button5 is bound to bound output roles: `DRS` & `PTP` 

I'm assuming with multiple paddle you've a set that shift normally, then add secondary mapping for BB+ or BB- to additional set of paddles and secondary map to gear up/down.

It sound it would be mentally tough to keep track of BB, so wouldn't recommend.

**Comment by Jeromebrivet** [comment]
No, i bind break bias x6 + for right paddle and break bias x6 - for left
paddle.

I use the 6th paddles of the wheel to do it.

This is what i do. This way i  Can play with the balance of the car between
high speed and low speed corner.

This function is directly available in my Rexing mayaris2 and it would be
great if we Can do the same with others wheels of the market and the
control mapper.

Le sam. 16 nov. 2024, 16:45, Steve Lewis ***@***.***> a
écrit :

**Comment by lewisst** [comment]
Sorry - I don't understand what you're asking for.

**Comment by Jeromebrivet** [comment]
Please look at it : https://rexing.eu/rexing-formula-wheel-mayaris-2/

- High quality USB spiral cable included
- OPTIONAL: Third set of paddles:

   - each with dual action. It’s possible to push it and pull it – with
   firm center position, providing a total of 4 configurable positions on just
   2 paddles.
   - additional two latching actions – pushing both paddles or pulling both
   paddles.
   - hall sensors
   - with rubber stops for reduced click sound
   - possibility to set multiple button presses for each paddle position,
   from 2 to 20 presses. Useful for brake bias, for example. Easily set up on
   the wheel, without additional software.
   - video example <https://www.youtube.com/watch?v=wkFyeeMiFJY&t=3s> of
   what the third paddles can be used for
   - price for optional third paddles: 120,00 Eur + VAT


Le dim. 17 nov. 2024 à 09:15, Steve Lewis ***@***.***> a
écrit :

**Comment by Jeromebrivet** [comment]
So on my wheel (in the native firmware) i Can choose multiplier
x3,x5,x6,x10,x20...

Le dim. 17 nov. 2024, 10:14, Jerome Brivet ***@***.***> a
écrit :
